build/Dockerfile

```diff
@@ -22,7 +22,9 @@ RUN echo "--- installing mizar dependencies ---" && \
     autoconf \
     automake \
     dh-autoreconf \
-    libtool
+    libtool \
+    wget
```

Comment:
please confirm if we need to add wget in this Docker environment, if yes, we will need to add it in build/Dockerfile.proxy also.

Reply:
I use wget to get .deb files for pulsar and I didn't add it to Dockerfile.proxy cause  it  already used wget while installing git.

Reply:
Got it. Thanks.